recipe/meta.yaml

```
@@ -10,21 +10,27 @@ source:
   url: https://cran.r-project.org/src/base/R-3/R-{{ version }}.tar.gz
   sha256: 0463bff5eea0f3d93fa071f79c18d0993878fd4f2e18ae6cf22c1639d11457ed
   patches:
-    - mac-unicode.patch  # [osx]
-    - 0002-Win32-Fix-trio-config-h-include-depth-issue.patch
-    - 0003-Win32-Do-not-static-libgcc-link.patch              # This patch is only correct if using MSYS2's tcltk.
-    - 0005-Win32-Extend-sqrt-NA_real_-hack-to-all-GCC-versions.patch
-    - 0006-Win32-GCC-5.3-Fix-ISNAN-int-emits-UD2-insn.patch
-    - 0007-Win32-MSYS2-ENV_CONV_EXCL_R_ARCH.patch
-    - 0008-Darwin-set-DYLD_FALLBACK_LIBRARY_PATH.patch
-    - 0009-R-3.3.0-Use-AC_SEARCH_LIBS-to-search-for-ncurses.patch
-    - 0011-Linux-Do-not-modify-LD_LIBRARY_PATH.patch
-    - 0012-macOS-include-cairo_h-not-cairo-xlib_h.patch
-    - 0013-javareconf-do-not-fail-on-compile-fail.patch
+    - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch
+    - 0002-Fix-trio-config.h-include-depth-issue.patch
+    - 0003-Win32-Do-not-link-static-libgcc.patch
+    - 0004-Win32-Extend-sqrt-NA_real_-hack-to-all-GCC-versions.patch
+    - 0005-Win32-Prevent-conversion-of-R_ARCH-to-abs-Windows-pa.patch
+    - 0006-Darwin-Avoid-setting-DYLD_FALLBACK_LIBRARY_PATH.patch
+    - 0007-Use-AC_SEARCH_LIBS-to-search-for-ncursesw-then-ncurs.patch
+    - 0008-Linux-Do-not-modify-LD_LIBRARY_PATH.patch
+    - 0009-macOS-include-cairo.h-not-cairo-xlib.h.patch
+    - 0010-javareconf-Do-not-fail-on-compile-fail.patch
+    - 0011-Revert-part-of-9b818c6dc00143ff18775a4015a3f43b5196f.patch
+    - 0012-javareconf-macOS-Continue-to-allow-system-Java-lt-9-.patch
+    - 0013-Allow-rpath-in-tcltk-dylib-LC_LOAD_DYLIB-command.patch
+    - 0014-Add-luuid-to-X_PRE_LIBS.patch
+    - 0015-link-Xt-to-uuid.patch
+    - 0016-Mimic-OpenBLAS-when-making-reference-Rblas-and-Rlapa.patch
+    - 0017-Foribly-remove-then-forcibly-mv-in-javareconf.in.patch
 
 build:
-  number: 1
-  merge_build_host: True   # [win]
+  merge_build_host: True  # [win]
+  number: 4
```

```suggestion
  number: 1004
```

We need this for the dual compiler parts